install.sh

```diff
@@ -11,32 +11,59 @@ waldo_exec2_name="sim_appcenter_build_and_upload.sh"
 waldo_asset1_name=
 waldo_asset2_name=
 
+waldo_exec_path=
+
 waldo_found_in_path=false
 waldo_is_reinstall=false
 
+function authorize() {
+    local _user_token=${WALDO_USER_TOKEN:-${TOKEN:-}}
+    local _auth_done=false
+    local _auth_status
+
+    if [[ -z $_user_token ]]; then
+        _user_token=$(find_user_token)
+    fi
+
+    if [[ -n $_user_token ]]; then
+        do_authorization $_user_token
+
+        _auth_status=$?
+    else
+        _auth_status=1
+    fi
+
+    if (( $_auth_status != 0 )); then
+        echo ""
+        echo "You are not currently authorized to use most Waldo CLI commands."
```

Comment:
authorized sounds a bit like 👮🏼‍♂️ you're not allowed no ? Maybe authenticated ?

Reply:
Perhaps. “authorize” and “authenticate” actually have 2 distinct meanings in the world of computer security (as you are no doubt aware) — and I always get them confused. I need to go re-read the definitions to convince myself that you are correct. Trust but verify.

Reply:
You are correct. `waldo auth` is definitely _authenticating_. I will need to update the help for that command in future pass. I will adjust the install message accordingly.